### PR TITLE
Disabled Providers

### PR DESCRIPTION
**Pull request by OshekharO**
I have disabled Providers which showing Link Not Found. ( Need Fixes )

### CODE REVIEW

**Comment by OshekharO** [comment]
Should I revert 2embed.org to 2embed.to ? It will start working if i will do this.

**Comment by LagradOst** [comment]
> Should I revert 2embed.org to 2embed.to ? It will start working if i do this.

YES!

**Comment by OshekharO** [comment]
> > Should I revert 2embed.org to 2embed.to ? It will start working if i do this.
> 
> YES!

Done, now the pr is ready to be merged.

**Comment by Blatzar** [comment]
:pensive: we'll try to fix the fuckery caused by removing history on master

**Comment by OshekharO** [comment]
> 

Alright thanks 😊

**Comment by Blatzar** [comment]
https://github.com/OshekharO/CloudStream-3/pull/1 No write access so cannot merge :pensive:

**Comment by OshekharO** [comment]
> [OshekharO#1](https://github.com/OshekharO/CloudStream-3/pull/1) No write access so cannot merge 😔

Now I think it's ready to be merged.

File: app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsAccount.kt [hunk 1]
```diff
@@ -79,33 +76,6 @@ class SettingsAccount : PreferenceFragmentCompat() {
...
                     val dialog = builder.show()
-
-                    val visibilityMap = mapOf(
-                        dialog.login_email_input to api.requiresEmail,
-                        dialog.login_password_input to api.requiresPassword,
-                        dialog.login_server_input to api.requiresServer,
-                        dialog.login_username_input to api.requiresUsername
-                    )
-
-                    if (context?.isTvSettings() == true) {
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsAccount.kt:90]: why did you remove this?

Comment (OshekharO) [reply app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsAccount.kt:90]: This is not removed by me. I think it's by mistakenly removed by Blatzar.

Comment (OshekharO) [reply app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsAccount.kt]: I have reverted the changes now it's good to go.
